tools/generate_tables_sql.php

```diff
@@ -45,7 +45,7 @@
     foreach($items AS $item){
         $paramId="";
         $bits=explode("{@}",trim($item));
-        if(ereg("Examiner[0-9]*" , $bits[1])){
+        if(preg_match("/Examiner[0-9]*/" , (string)(array_key_exists(1,$bits) ? $bits[1] : null))){
             continue;
```

Comment:
I would do:
`if ( !empty($bits[1]) && preg_match("/Examiner[0-9]*/" , $bits[1]) ) `

I find it a bit simpler/easier to understand.